pkg/ebpf/events_pipeline.go

```diff
@@ -31,7 +31,7 @@ func (t *Tracee) handleEvents(ctx context.Context) {
 	var errcList []<-chan error
 
 	// Source pipeline stage.
-	eventsChan, errc := t.decodeEvents(ctx, t.eventsChannel)
+	eventsChan, evtPool, errc := t.decodeEvents(ctx, t.eventsChannel)
```

Comment:
WDYT about moving the pool to the tracee struct so it won't need to be passed per pipeline stage? Later when #3254 is tackled, the pool should belong to the pipeline.

Reply:
Good idea. I've just did that. Pushing changes soon.